### DISCUSSION TITLE
Is Wazuh Agent for Kubernetes in plans?

### DISCUSSION THREAD

**Discussion by ALL-SPACE-Anas** [wazuh/wazuh-kubernetes#887] [General]
I've been interested and working towards deploying Wazuh Kubernetes with the primary purpose of monioring, vulnerability scanning and container security for kubernetes cluster. I've just realized that Wazuh doesn't offer agents for Kubernetes, which I find odd. But anyways, is this something in the works for future releases?

**Comment by zakkg3** [comment]
check this out

https://github.com/zakkg3/wazuh-agent-container/
